### DISCUSSION TITLE
如何正确关闭ipv6/正确配置ipv6

### DISCUSSION THREAD

**Discussion by github-h** [coolsnowwolf/lede#9540] [Q&A / 快速问答] [answered]
如题，如果并不需要应该怎么设置是正确关闭啊

或者如果需要的话应该怎么进行设置 网上乱七八糟的教程太多了，麻烦大雕能给讲一个推荐设置方法
![图片](https://user-images.githubusercontent.com/49114731/173229262-85f6f762-8f06-44f0-840a-671c7f05d944.png)

**Comment by shampan2022** [comment]
关闭IPV6可参考，已实测。https://www.cnblogs.com/NueXini/p/15707858.html

**Comment by chpamhe** [comment]
wan接口：关闭获取IPv6地址的方式，修改成已禁用（PPOE协议模式下）。取消勾选：使用内置的IPv6管理。
LAN接口：取消勾选：使用内置的IPv6管理，DHCP服务器IPv6设置里面，禁用：路由通告服务、DHCPv6服务、NDP代理。
再返回网络接口菜单，全局网络选项 IPv6 ULA 前缀  这里清空掉。保存并应用。观察 网络接口页面下，是否还有240开头的IPv6地址，或者是本地的IPv6地址，如果还有重启一下路由器。

**Comment by github-h** [comment, marked as answer]
在这里补一个大雕的推荐方法
![图片](https://user-images.githubusercontent.com/49114731/173240580-b36ba7ff-4b46-4bd8-944a-c42fbb0cf389.png)

**Reply by github-h** [reply]
大雕这个方法确实可行
![图片](https://user-images.githubusercontent.com/49114731/173245024-0499103f-b726-4a87-9ca2-d0167e682b31.png)

没有24开头的地址了
如果以上操作过后还是有的记得重启下设备

**Reply by github-h** [reply]
同 #9583
